src/Network/Network/ApplicationGateway/RewriteRule/AzureApplicationGatewayRewriteRuleBase.cs

```diff
@@ -33,6 +33,16 @@ public class AzureApplicationGatewayRewriteRuleBase : NetworkBaseCmdlet
         [ValidateNotNullOrEmpty]
         public PSApplicationGatewayRewriteRuleActionSet ActionSet { get; set; }
 
+        [Parameter(
+        Mandatory = false,
+        HelpMessage = "The rule ordering of this rewrite rule in the rewrite rule set")]
+        public int RuleSequence { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Condition for the rewrite rule to execute")]
+        public List<PSApplicationGatewayRewriteRuleCondition> Condition { get; set; }
```

Comment:
@sobaner please make this `PSApplicationGatewayRewriteRuleCondition[]` instead of `List<PSApplicationGatewayRewriteRuleCondition>`